### DISCUSSION TITLE
Practical 3 - Table 2

### DISCUSSION THREAD

**Discussion by rachael-clark** [IBAHCM/RPiR#430] [Old]
Rachael Clark - https://github.com/IBAHCM/RachaelClarkSeries03.git

**Comment by Sambainbridge** [comment]
Sambainbridge-https://github.com/IBAHCM/githubSambainbridgeSeries03.git

**Reply by JumaMinya** [reply]
Juma Minya: https://github.com/IBAHCM/JumaMinyaSeries03.git

**Comment by DylanCarbone** [comment]
DylanCarbone: https://github.com/IBAHCM/DylanCarboneSeries03.git
